app/core/utils.coffee

```diff
@@ -382,6 +382,23 @@ module.exports.findNextLevel = (levels, currentIndex, needsPractice) ->
 module.exports.needsPractice = (playtime=0, threshold=2) ->
   playtime / 60 > threshold
 
+module.exports.sortCourses = (courses) ->
+  orderedIDs = [
+    '560f1a9f22961295f9427742' # introduction-to-computer-science
+    '5632661322961295f9428638' # computer-science-2
+    '5789587aad86a6efb573701e' # game-development-1
+    '5789587aad86a6efb573701f' # web-development-1
+    '56462f935afde0c6fd30fc8c' # computer-science-3
+    '57b621e7ad86a6efb5737e64' # game-development-2
+    '5789587aad86a6efb5737020' # web-development-2
+    '56462f935afde0c6fd30fc8d' # computer-science-4
+    '569ed916efa72b0ced971447' # computer-science-5
```

Comment:
Might be better to set these as exports of an IDs module, so that they can all be looked up in one place.

```
courseIDs = require('core/ids')

...
orderedIDs = [
  courseIDs.INTRODUCTION_TO_COMPUTER_SCIENCE,
  courseIDs.COMPUTER_SCIENCE_2,
  ...
]
```

Something along those lines.